ga4gh/client.py

```diff
@@ -38,10 +36,13 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         else:
             logLevel = logging.DEBUG
         self._logger.setLevel(logLevel)
-        handler = logging.StreamHandler()
-        formatter = logging.Formatter('%(name)s: %(message)s')
-        handler.setFormatter(formatter)
-        self._logger.addHandler(handler)
+
+        requestsLog = logging.getLogger("requests.packages.urllib3")
```

Comment:
I've put in a TODO here. I'm not particularly happy with what we've ended up with --- I wish I understood what was actually happening here. It'll do for now though I think.